File: a2oj/ladder_11/038_little_elephant.cpp
// https://codeforces.com/problemset/problem/205/A

#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n = 0, mini = INT_MAX, cnt = 0, found_at = 0;
    cin >> n;
    for (int i = 1; i <= n; i++) {
        int temp = 0;
        cin >> temp;
        if (temp < mini) {
            mini = temp;
            found_at = i;
            cnt = 1;
        } else if (temp == mini) {
            cnt += 1;
        }
    }
    if (cnt > 1) {
        cout << "Still Rozdil";
    } else {
        cout << found_at;
    }
    return 0;
}

File: a2oj/ladder_11/038_little_elephant.py
# https://codeforces.com/problemset/problem/205/A

n = int(input())
towns = list(map(int, input().split()))
mini = 1000000001
found_at = 0
cnt = 0
for i, town in enumerate(towns):
    if town < mini:
        cnt = 1
        mini = town
        found_at = i
    elif town == mini:
        cnt += 1
if cnt > 1:
    print("Still Rozdil")
else:
    print(found_at+1)
